Python/pywarpx/picmi.py

```diff
@@ -73,6 +73,10 @@ class Species(picmistandard.PICMI_Species):
         Whether or not to deposit the charge and current density for
         for this species
 
+    warpx_random_theta: bool, default=True
+        Whether or not to add random angle to the particles in theta
+        when in 2D-RZ mode.
```

Comment:
We usually simply call this mode RZ in WarpX.
```suggestion
        when in RZ mode.
```